pkg/timer/tablestore/sql_test.go

```diff
@@ -737,11 +736,10 @@ func TestWithSession(t *testing.T) {
 		sctx.On("ExecuteInternal", matchCtx, "SET @@time_zone=%?", mock.Anything).
 			Return(nil, errors.New("SET tz error")).
 			Once()
-		sctx.On("Close").Once()
 		mockCb1.MethodCalled("cb1", se)
 		return nil
 	}))
-	require.True(t, pool.se.IsInternalClosed())
+	require.True(t, pool.se.IsAvoidReuse())
```

Comment:
This behavior is guarded by a test. I'm not sure why it should close internally inside the `withSession` 🤔 . @lcwangchao PTAL.

Reply:
I think `withSession` think `pool.Put` will discard a closed session without using it. But in a test mock pool, it may not. It's better to use `AvoidToResuse` than `Close`